packages/common/CHANGELOG.md

```diff
@@ -19,13 +31,76 @@ import Common from '@ethereumjs/common'
 const common = new Common({ chain: 'mainnet', hardfork: 'muirGlacier' })
 ```
 
-### Hardfork by block number
+### EIP Support
+
+EIPs are now native citizens within the `Common` library, see PRs [#856](https://github.com/ethereumjs/ethereumjs-vm/pull/856), [#869](https://github.com/ethereumjs/ethereumjs-vm/pull/869) and [#872](https://github.com/ethereumjs/ethereumjs-vm/pull/872). Supported EIPs have their own configuration file like the [eips/2537.json](https://github.com/ethereumjs/ethereumjs-vm/blob/master/packages/common/src/eips/2537.json) file for the BLS precompile EIP and EIP settings can be activated by passing supported EIP numbers to the constructor:
+
+```typescript
+const c = new Common({ chain: 'mainnet', eips: [2537] })
+```
+
+The following EIPs are initially supported within this release:
+
+- [EIP-2537](https://eips.ethereum.org/EIPS/eip-2315) BLS Precompiles
+- [EIP-2315](https://eips.ethereum.org/EIPS/eip-2315) EVM Subroutines (PR [#876](https://github.com/ethereumjs/ethereumjs-vm/pull/876))
+
+EIPs provided are then activated and parameters requested with `Common.param()` being present in these EIPs take precedence over the setting from the latest hardfork.
+
+There are two new utility functions which dedicatedly return the hardfork respectively EIP values:
```

Comment:
```suggestion
There are two new utility functions which return hardfork and EIP values respectively:
```

Reply:
Will mark as resolved along fixing locally and push once finished on all the change requests.